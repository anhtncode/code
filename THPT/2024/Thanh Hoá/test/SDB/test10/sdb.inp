10
638664 820709
10 561503
3 297443
650487 789407
3 446970
7 182809
4 846391
7 921407
4 178371
4 789104
