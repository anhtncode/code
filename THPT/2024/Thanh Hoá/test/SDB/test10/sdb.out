6
53
42
6
47
34
62
66
34
62
